CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -399,157 +424,94 @@ public void Destroy() {
     }
     
     
-    private void DestroyCommandBuffers() {
-        if (null != commandBuffers && commandBuffers.size() > 0) {
-            commandBuffers.forEach(el -> {el.Destroy();});
-            commandBuffers.clear();
-            commandBuffers = null;
-        }       
-    }
-    
-    private void DestroyVertexBuffer() {
-        if (null != cvkVertexBuffer) {
-            cvkVertexBuffer.Destroy();
-            cvkVertexBuffer = null;
-        }
-    }
-    
-    private void DestroyUniformBuffers() {
-        if (vertexUniformBuffers != null) {
-            vertexUniformBuffers.forEach(el -> {el.Destroy();});
-            vertexUniformBuffers = null;
-        }
-    }
-    
-    private void DestroyDescriptorSets(){
-        if (pDescriptorSets != null) {
-            vkFreeDescriptorSets(cvkDevice.GetDevice(), cvkSwapChain.GetDescriptorPoolHandle(), pDescriptorSets);
-            pDescriptorSets = null;
-        }
-    }
-    
-    private void DestroyPipeline() {     
-        if (pipelines != null) {
-            for (int i = 0; i < pipelines.size(); ++i) {
-                vkDestroyPipeline(cvkDevice.GetDevice(), pipelines.get(i), null);
-                pipelines.set(i, VK_NULL_HANDLE);
-            }
-            pipelines.clear();
-            pipelines = null;
-        }       
-    }
+    // ========================> Swap chain <======================== \\
     
-    private void DestroyPipelineLayouts() {
-        if (pipelineLayouts != null) {
-            for (int i = 0; i < pipelineLayouts.size(); ++i) {
-                vkDestroyPipelineLayout(cvkDevice.GetDevice(), pipelineLayouts.get(i), null);
-                pipelineLayouts.set(i, VK_NULL_HANDLE);
+    /**
+        Called just before the swapchain is about to be destroyed allowing the
```

Comment:
Comment block format is not standard

Reply:
Changed.  Btw that was one you added when you refactored RecreateSwapchain ;)